### DISCUSSION TITLE
Album art in bottom bar glitch: green flicker

### DISCUSSION THREAD

**Discussion by bartkl** [jcorporation/myMPD#971] [General]
myMPD version: 10.2.1

**Describe the bug**
In Chrome there's a glitch with the small displayed album art in the bottom bar. It flickers and is covered (in part) in green. Note that if I change the highlight color of the UI theme, the color in the glitch remains green.

This issue does not occur in Firefox, so it probably has to do with my Chrome and system configuration/setup. I realize it's not a bug with myMPD so much, but maybe more users are having this problem and we can help each other out.

Curiously, I don't seem to get this issue anywhere else! All other images are fine, including the cover art in different views in the player.

I did use to see glitchy favicons for some sites (YouTube and Reddit come to mind), but I fixed those with forcing a sRGB color profile.

**To Reproduce**
Simply observe the small displayed album art in the bottom bar of the player (see attached screencast).

**Expected behavior**
The album art should not flicker green.

**Screenshots**
[Screencast from 02-08-2023 07:35:50 PM.webm](https://user-images.githubusercontent.com/2497867/217621632-a8bc2473-4a8c-4a3f-88b4-36a4adc359ae.webm)

**Server plattform (please complete the following information):**
 - OS: Raspbian
 - OS version: 10
 - Platform: armhf (Raspberry Pi 3B+)
 - MPD version: 0.23.12

**Client plattform (please complete the following information):**
 - Device: Laptop
 - OS: Pop!_OS 22.04 LTS (Ubuntu based)
 - Browser: Chrome
 - Browser version: Version 110.0.5481.77 (Official Build) (64-bit)

**Debug logs (please attach if it can be useful)**

Look at https://jcorporation.github.io/myMPD/debug

 - myMPD debug log: n/a
 - MPD verbose log: n/a
 - Browser error console: nothing
 - Browser network console: n/a
 
**Configuration (please attach if it can be useful)**
n/a

**Comment by jcorporation** [comment]
As this is not a myMPD bug, I move it to discussions.

**Comment by jcorporation** [comment]
> Note that if I change the highlight color of the UI theme, the color in the glitch remains green.

The mympd image as cover placeholder is always green and not in the highlight color, because it can not styled with css.
